src/Components/Posts.js

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+const Posts = () => {
+  return <div>Posts</div>;
+};
+
+export default Posts;
```

Comment:
```suggestion
import React from "react";
import Post from "./Post";

const Posts = () => {
  return (<div>
  <Post />
  <Post />
  <Post />
  </div>);
};

export default Posts;
```

Instead of putting four single `<Post />` components inside of your App component, you should put them inside of your Posts component. The same is true for Contacts and `<ContactUserCard />` . I won't refactor App.js, but if you put each `<Post />` here, you no longer have to put your `<Post />` components inside of App.js.

Reply:
Thanks for the feedback, Sam! So is it generally more optimal to nest components this way?